documents/knowledge/elements.yaml

```
@@ -1149,6 +1149,8 @@
   arity: 2
   overloads:
     any-any: a.replace(b,"")
+    num-fun: first a positive integers where b is truthy
+    fun-num: first b positive integers where a is truthy
```

```suggestion
    num-fun: first a positive integers where b is truthy
    fun-num: first b positive integers where a is truthy
    any-any: a.replace(b,"")
```
It just seems like the any-any overloads is matched before the num-fun/fun-num overloads right now.

---

What do you think of a file containing the order of overloads?

---

`elements.yaml` has that already